src/omv/framebuffer.h

```diff
@@ -36,4 +38,8 @@ static struct jpegbuffer {
 
 // Use this macro to get a pointer to the free SRAM area located after the framebuffer.
 #define FB_PIXELS() (MAIN_FB()->pixels+MAIN_FB_SIZE())
+
+// Transfers the frame buffer to the jpeg frame buffer if not locked.
+void copy_fb_to_jpeg_fb();
```

Comment:
If you're going to make framebuffer.c, please prefix fb functions (ex. fb_ or framebuffer_).
Note this function is mislabeled and misused (will explain more in comment on sensors.c). I think it should be called fb_update_jpeg_buffer().